src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,17 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User nativeUser = userService.findByEmail(email);
+
+        if (nativeUser == null) {
+            return false;
+        } else {
+            return nativeUser.getEmail().equals(email)
+                    && nativeUser.getPassword().equals(password);
+        }
```

Comment:
Do you need to check user's email if it isn't null? You don't need `else` here
```suggestion
        } 
        return nativeUser.getPassword().equals(password);
```